.github/workflows/release-binaries.yml

```diff
@@ -262,6 +262,8 @@ jobs:
         else
           cargo build --bin ockam --target ${{ matrix.target }} --verbose --release
         fi
+      env:
+        RUSTFLAGS: '-C target-feature=+crt-static'
```

Comment:
🤔 wonder if this being set for all cases in the build matrix has unintended effects on the macos binaries.

Would be better to set it only for the linux targets.

Reply:
Updated.